init-openstack.html.md.erb

```diff
@@ -151,28 +151,36 @@ You must create and configure two Security Groups to restrict incoming network t
     $ git clone https://github.com/cloudfoundry/bosh-deployment
 
     # Fill below variables (replace example values) and deploy the Director
+    $ DIRECTOR_NAME='bosh-1'     # arbitrary name for the director
```

Comment:
how about putting comments inline? i think extra indirection thru vars is not really helpful.